### PR TITLE
Scorecard Authored

### PR DESCRIPTION
**Pull request by samirparhi-dev**
Scorecard Authored Please review

### CODE REVIEW

File: playbooks/besman-scorecard-playbook.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    echo -e "\n${BLUE}🛫 Preparing your machine for ScoreCard Run .${RESET}"
+    # Specify the directory to save the scan results
+    mkdir -p "$HOME/$PROJECT_NAME/scoreCard"
+
+}
+
+function __besman_execute {
+
+    # creating a CodeQL database for Java code
+    curl -X 'GET' \
```
Comment (sand-hya) [inline playbooks/besman-scorecard-playbook.sh:40]: @samirparhi-dev for this scorecard playbook to work, github actions for scorecard has to be run

**Review by sand-hya — CHANGES_REQUESTED** [review]
@samirparhi-dev for this scorecard playbook to work, github actions for scorecard has to be run

File: playbooks/besman-scorecard-playbook.sh (hunk 1 above)
Comment (samirparhi-dev) [reply playbooks/besman-scorecard-playbook.sh:40]: @sand-hya : Noted

**Review by pramit-d — CHANGES_REQUESTED** [review]
@samirparhi-dev Please add a version for playbooks. The file name should contain the version number.
Use [sbom generator](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbooks/besman-spdx-sbom-generator-0.0.1-playbook.sh) playbook as an example and modify your playbook file accordingly. 

Remove codeql, fossology, snyk, sonar-scanner playbooks from this PR as they are already raised in another PRs.

Also, add your playbook data in [playbook-metadata.json](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbook-metadata.json) file.

**Comment by samirparhi-dev** [comment]
> @samirparhi-dev Please add a version for playbooks. The file name should contain the version number. Use [sbom generator](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbooks/besman-spdx-sbom-generator-0.0.1-playbook.sh) playbook as an example and modify your playbook file accordingly.
> 
> Remove codeql, fossology, snyk, sonar-scanner playbooks from this PR as they are already raised in another PRs.
> 
> Also, add your playbook data in [playbook-metadata.json](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbook-metadata.json) file.

I am currently working on the new playbooks, will upload that soon.

**Pull request closed without merging**